features/buyer/requirements.feature

```diff
@@ -136,14 +136,20 @@ Scenario: Create user research participants
    And I don't see the 'Review and publish your requirements' link
 
 @copy
```

Comment:
If we plan to release this before moving our buyer app through to staging (which I would encourage) then we should tag this test to exclude it from staging